ports/stm/peripherals/stm32l4/clocks.c

```diff
@@ -57,7 +59,7 @@ void stm32_peripherals_clocks_init(void) {
 
     // Configure LSE Drive
     HAL_PWR_EnableBkUpAccess();
-    __HAL_RCC_LSEDRIVE_CONFIG(RCC_LSEDRIVE_LOW);
+    __HAL_RCC_LSEDRIVE_CONFIG(BOARD_LSE_DRIVE_LEVEL);
```

Comment:
I don't see any default value for `BOARD_LSE_DRIVE_LEVEL` defined anywhere. There should be a default or all the other STM32L4 boards will fail to compile.

Reply:
It is defined in `ports/stm/boards/swan_r5/mpconfigboard.h` since it is a board-specific property. Adding a default is potentially dangerous since it depends on the characteristics of the baord.

At present, only the Swan R5 board is based on the L4.  As new boards are added, they will need to define the correct drive level that is calculated from the board characteristics.  If a new board doesn't define this symbol, the [error message produced](https://github.com/adafruit/circuitpython/pull/7663/files#diff-050e9e1da1d1fa930cd8a8dfd35421574b88d95c5ea452152306f8246b0fc943R43) describes where to look to determine the correct value.

Reply:
Would a high drive level be safe? How did you determine the level you needed?

Reply:
Or is high drive potentially damaging to some crystals?

Reply:
What I am thinking about is that if a board is based on a reference design with the reference crystal, the drive level is known. I see that F7 and H4 boards also have a default of "low". It sounds like your board is different. But a vanilla board could work with "low", if that's what's fine for the reference design. Otherwise someone starting to port a new board will have to figure this out. That's why I'd rather have a safe default. If it doesn't work, then they can change it.

Reply:
Unfortunately, finding out that it doesn't work can happen quite far down the road, as was our case. (And is mentioned specifically in the opening of the [application note](https://www.st.com/resource/en/application_note/cd00221665-oscillator-design-guide-for-stm8afals-stm32-mcus-and-mpus-stmicroelectronics.pdf).  I didn't personally do the calculations needed so can't answer how `MEDIUMLOW` was arrived at, but it was found to give better stability and is something specific to the Swan R5 board, as it is for all boards I believe since it's inherently tied to the capacitance involved.  Using `LOW` as the default is just a best guess. Board designers using an LSE are best advised to follow through the documentation mentioned in the comments to determine the correct value.

If you feel a default value is desirable despite these caveats I of course can add it.

Reply:
I'm willing to go along with this, since there are no other boards affected.

We have had our own issues with crystal oscillator parameters on the RP2040 -- on a small number of samples, the xtal osc can take much longer to start up than might be expected. There is a settable delay for this. I don't think there is a drive setting on that chip.